extmod/network_wiznet5k.c

```diff
@@ -766,7 +768,13 @@ STATIC mp_obj_t wiznet5k_make_new(const mp_obj_type_t *type, size_t n_args, size
 STATIC mp_obj_t wiznet5k_regs(mp_obj_t self_in) {
     (void)self_in;
     printf("Wiz CREG:");
+    #if _WIZCHIP_ == 5500
```

Comment:
Rather than duplicating the for loop 3 times, can you do

```
    #if _WIZCHIP_ == 5500
    const int nregs = 0x50;
    #elif _WIZCHIP_ == 5105
    const int nregs = 0x90;
    #else
    const int nregs = 0x60;
    #endif
    for (int i = 0; i < nregs; ++i) {
```

Reply:
Rather than having the `#else` be 0x60 -- can the default be the lower number, and explicitly set it to the higher number when it's known to be a chip with more registers?